static/bidder-info/yieldmo.yaml

```diff
@@ -1,6 +1,11 @@
 maintainer:
   email: "prebid@yieldmo.com"
 capabilities:
+  app:
+    mediaTypes:
+    - banner
+    - video
   site:
     mediaTypes:
     - banner
+    - video
```

Comment:
The yeildmo adapter is hardcoded to support banner types only. You'll need to add auction type detection and update the bid response type.

```
bidResponse.Bids = append(bidResponse.Bids, &adapters.TypedBid{
	Bid:     &sb.Bid[i],
	BidType: "banner",
})
```

Please also add json test cases for the new types (site video, app banner, and app video).

Reply:
Thank you for pointing this out. My mistake. I have made the necessary changes, and added the json test cases you recommended.

Reply:
Great. Thank you. Looking good now.